src/LidoDAO.ts

```diff
@@ -22,12 +23,24 @@ export function handleSetApp(event: SetAppEvent): void {
       // updating only in case of a new contract address
       if (entity.impl != event.params.app) {
         const repo = AppRepo.bind(Address.fromString(repoAddr))
-        const latest = repo.getLatestForContractAddress(event.params.app)
-        const semVer = latest.getSemanticVersion()
 
-        entity.major = semVer[0]
-        entity.minor = semVer[1]
-        entity.patch = semVer[2]
+        const triedLatest = repo.try_getLatestForContractAddress(event.params.app)
+        if (triedLatest.reverted) {
+          if (network !== 'holesky') {
+            throw new Error('indexing error - getLatestForContractAddress method is reverted')
```

Comment:
lets add a reason if exist

Reply:
there is not error reason in that object